.travis/linux_openresty_mtls_runner.sh

```diff
@@ -109,13 +109,34 @@ script() {
     sleep 1
     cat logs/error.log
 
-
+    # correct certs 
     code=$(curl -i -o /dev/null -s -w %{http_code}  --cacert ./t/certs/mtls_ca.crt --key ./t/certs/mtls_client.key --cert ./t/certs/mtls_client.crt -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' https://admin.apisix.dev:9180/apisix/admin/routes)
     if [ ! $code -eq 200 ]; then
         echo "failed: failed to enabled mTLS for admin"
         exit 1
     fi
 
+    # no certs
+    code=$(curl -i -o /dev/null -s -w %{http_code} -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' https://admin.apisix.dev:9180/apisix/admin/routes)
+    if [ ! $code -eq 000 ]; then
```

Comment:
Does `000` equal to `0`?

Reply:
I think it's a default value when `curl` got an empty respond.

Reply:
It caused exiting that make travis failed. So I removed them ..